### ISSUE TITLE
Possible issue with forward search and the subfiles package

### ISSUE THREAD

**Issue by wbthomason** [latex-lsp/texlab#208]
# Bug description
When using forward search in a multi-file document with sections managed using https://ctan.org/pkg/subfiles, `texlab` seems to use the wrong path for substituting the `%p` variable in invoking the viewer for forward search.

In particular: using forward search from the main LaTeX file succeeds, indicating that the basic forward-search setup is configured correctly. However, trying forward search from a subfile results in an error when the viewer tries to open a nonexistent file.

For example, in a project organized as follows:
```
main.tex
sections/
    foo.tex
```
where `foo.tex` is included in `main.tex` using `\subfile{sections/foo.tex}`, forward search from a point in `foo.tex` causes the viewer to try to open `sections/./foo.pdf` instead of `main.pdf`.

# Versions
**texlab version**: 1.10.0
**LSP client/editor**: `coc.nvim` and Neovim, both latest from `master`.
**Platform**: Arch Linux

# Config

```json
"latex.build.onSave": false,
"latex.forwardSearch.executable": "okular",
"latex.forwardSearch.args": ["--unique", "file:%p#src:%l%f"],
```

Empty `latexmkrc`, compilation managed with `vimtex` configured as
```vim
let g:vimtex_view_forward_search_on_start = 0
let g:vimtex_view_general_viewer = 'okular'
let g:vimtex_view_general_options = '--unique file:@pdf\#src:@line@tex'
let g:vimtex_view_general_options_latexmk = '--unique'
```

**Comment by pfoerster** [comment]
Thanks for the detailed report. The issue should be fixed with `v2.0.0`.

**Comment by wbthomason** [comment]
Confirmed fixed!
